cmake/FindFilesystem.cmake

```diff
@@ -180,12 +180,17 @@ if(CXX_FILESYSTEM_HAVE_FS)
 
         int main() {
             auto cwd = @CXX_FILESYSTEM_NAMESPACE@::current_path();
-            return static_cast<int>(cwd.string().size());
+            for (auto &_ : @CXX_FILESYSTEM_NAMESPACE@::directory_iterator{cwd}) {
+                (void) _;
+            }
+            auto root = cwd.root_directory();
+            (void) root;

```

Comment:
What do these `(void) ...` do?

Reply:
They avoid unused-variable-warnings. Not really necessary here but just good style.